### PR TITLE
[4.12-4.13] gcp-ipi-workers-marketplace jobs

### PR DESCRIPTION
**Pull request by jianli-wei**
- add periodic ci jobs `gcp-ipi-workers-marketplace-f28` and `gcp-ipi-workers-marketplace-f28-destructive` in 4.13 & 4.12 amd64 nightly

### CODE REVIEW

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-gcp-ipi-workers-marketplace-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-gcp-ipi-workers-marketplace-f28

**Comment by gpei** [comment]
cc @liangxia to review

**Comment by liangxia** [comment]
There are conflicts.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -2184,6 +2184,29 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-sdn
+- as: gcp-ipi-workers-marketplace-f28
+  cron: 15 15 13 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2193]: You might want to add `and not @fips` to exclude fips testing as the profile does not enable fips.

**Pull request merged**